src/main/java/ch/njol/skript/expressions/ExprElement.java

```diff
@@ -29,89 +29,142 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.skript.util.LiteralUtils;
+import ch.njol.skript.util.Patterns;
 import ch.njol.util.Kleenean;
 import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
 import com.google.common.collect.Iterators;
+import org.apache.commons.lang.ArrayUtils;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 import java.lang.reflect.Array;
 import java.util.Iterator;
 
-@Name("Element of")
-@Description({"The first, last or a random element of a set, e.g. a list variable.",
-		"See also: <a href='#ExprRandom'>random</a>"})
-@Examples("give a random element out of {free items::*} to the player")
-@Since("2.0, 2.7 (relative to last element)")
-public class ExprElement extends SimpleExpression<Object> {
+@Name("Elements")
+@Description({
+		"The first, last, range or a random element of a set, e.g. a list variable.",
+		"See also: <a href='#ExprRandom'>random</a>"
+})
+@Examples("broadcast the first 3 elements of {top players::*}")
+@Since("2.0, 2.7 (relative to last element), INSERT VERSION (range of elements)")
+public class ExprElement<T> extends SimpleExpression<T> {
+
+	private static final Patterns<ElementType[]> PATTERNS = new Patterns<>(new Object[][]{
+		{"[the] (first|1:last) element [out] of %objects%", new ElementType[] {ElementType.FIRST_ELEMENT, ElementType.LAST_ELEMENT}},
+		{"[the] (first|1:last) %integer% elements [out] of %objects%", new ElementType[] {ElementType.FIRST_X_ELEMENTS, ElementType.LAST_X_ELEMENTS}},
+		{"[a] random element [out] of %objects%", new ElementType[] {ElementType.RANDOM}},
+		{"[the] %integer%(st|nd|rd|th) [1:[to] last] element [out] of %objects%", new ElementType[] {ElementType.ORDINAL, ElementType.TAIL_END_ORDINAL}},
```

Comment:
It works if you surround it with parentheses. It seems to be an issue with the parser itself, rather than the element's pattern. I think it's out of the scope of this pull request.